yeezyCopper.js

```diff
@@ -60,69 +64,153 @@ function pay () {
     }
   });
 
-  $('input#dwfrm_payment_creditCard_cvn').val(function () {
+  $('input#dwfrm_payment_creditCard_cvn').val(function() {
     return config.security_code;
   });
 
-  setTimeout(function () {
+  setTimeout(function() {
     placeOrder();
   }, 300);
 }
+
+function _x(STR_XPATH) {
```

Comment:
I copied this function from someone I reached out to for help, and they said this would help me click the overnight shipping button by referencing a Xpath instead of an element ID, so I kept this. Not sure how else to rewrite this, so should I keep it? I think I fixed the parameter name to camelCase. Thanks for explaining how all caps works for parameters -- that's going to be helpful to know.
<img width="638" alt="screen shot 2016-06-15 at 9 14 51 pm" src="https://cloud.githubusercontent.com/assets/5168320/16102553/7ecdcee0-333e-11e6-92ca-3ae35d3e1bf5.png">